frontend/src/providers/AuthProvider.tsx

```diff
@@ -20,16 +25,40 @@ export const AuthContext = createContext<AuthContextType>({
   authenticate: () => {},
```

Comment:
why this method is empty?

Reply:
Those are default values for functions passed within react context. I learned that this is good practise to keep initial values for functions that will be passed empty. Please correct me if I am wrong.
Implementation of this function can be found later in this file.


```typescript
 /**
   * Function to authenticate the user.
   *
   * This function sets the user as authenticated, redirects to the logged-in
   * page, and displays a toast message.
   */
  const authenticate = () => {
    setIsAuthenticated(true);
    navigate("/logged");
    handleToastOpening("Logged in.");
  };
  `````